be/src/common/config.h

```diff
@@ -902,6 +902,11 @@ CONF_mInt64(l0_l1_merge_ratio, "10");
 CONF_mInt64(l0_max_file_size, "209715200"); // 200MB
 CONF_mInt64(l0_max_mem_usage, "67108864");  // 64MB
 CONF_mInt64(max_tmp_l1_num, "10");
+CONF_mBool(enable_parallel_get_and_bf, "false");
+CONF_mInt64(min_enable_bf_kv_num, "100000000");
+CONF_mInt32(page_shard_read_ratio, "20");
+CONF_mInt32(min_get_kv_thread_num, "1");
+CONF_mInt32(max_get_kv_thread_num, "10");
```

Comment:
Why the memory will be double? I think the bloom filter should be small than original data.

Reply:
> Why the memory will be double? I think the bloom filter should be small than original data.

Because we only keep a small memory in L0 without bloom filter. If we enable bloom filter for persistent index, we will create a bloom filter for each L1 and the bloom filters will cost some memory because the origin data size of L1 maybe very large.